performance/reports/JsonBenchmark.7.0.0-pre.md

```diff
@@ -0,0 +1,53 @@
+﻿```
+
+BenchmarkDotNet v0.14.0, Ubuntu 24.04.1 LTS (Noble Numbat)
+AMD EPYC 7763, 1 CPU, 4 logical and 2 physical cores
+.NET SDK 9.0.101
+  [Host]     : .NET 9.0.0 (9.0.24.52809), X64 RyuJIT AVX2
+  DefaultJob : .NET 9.0.0 (9.0.24.52809), X64 RyuJIT AVX2
+
+
+```
+| Method          | FileName    | Mean         | Error      | StdDev     | Ratio | RatioSD | Gen0     | Gen1     | Gen2    | Allocated | Alloc Ratio |
+|---------------- |------------ |-------------:|-----------:|-----------:|------:|--------:|---------:|---------:|--------:|----------:|------------:|
+| **Farkle6String**   | **big.json**    |   **811.523 μs** |  **2.7956 μs** |  **2.4782 μs** |  **1.00** |    **0.00** |  **19.5313** |        **-** |       **-** |  **338793 B** |       **1.000** |
+| Farkle7String   | big.json    |   722.303 μs |  3.3911 μs |  3.1720 μs |  0.89 |    0.00 |        - |        - |       - |       1 B |       0.000 |
+| PidginString    | big.json    | 2,196.839 μs |  7.9004 μs |  6.5972 μs |  2.71 |    0.01 |        - |        - |       - |    6086 B |       0.018 |
+| IronyString     | big.json    | 2,908.325 μs | 27.7690 μs | 25.9752 μs |  3.58 |    0.03 | 179.6875 | 132.8125 | 42.9688 | 3086216 B |       9.109 |
```

Comment:
Now that Farkle is a C# library I wanted to compare it to other C# parsers, but both turned out to be quite slower. Benchmarks for Pidgin and Irony will be removed after the 7.0.0 release.